tests/models.ts

```diff
@@ -67,3 +67,35 @@ export type ChannelSelector = 'Player' | 'Webamp';
 export type UserType = 'privs' | 'patron' | 'no-login';
 
 export type PageType = 'radio' | 'tv';
+
+export enum FacetGroupFilterHeaderEnum {
+  YEAR_PUBLISHED = 'Year Published range filter',
+  MEDIATYPE = 'Media Type filters',
+  YEAR = 'Year filters',
+  SUBJECT = 'Subject filters',
+  COLLECTION = 'Collection filters',
+  CREATOR = 'Creator filters',
+  LANGUAGE = 'Language filters',
+  AVAILABILITY = 'Availability filters',
+  PART_OF = 'Part Of', // this only appears in collection page
+}
+
+export const CommonFacetGroupHeaders = [
+  'Year Published range filter',
+  'Media Type filters',
+  'Year filters',
+  'Subject filters',
+  'Collection filters',
+  'Creator filters',
+  'Language filters',
```

Comment:
QUESTION: Do these names need to match up with the constant values defined in `FacetGroupFilterHeaderEnum`? If so, perhaps better to use those defined constant values here, rather than repeat the literal string (which could be prone to typo errors). Same feedback for `SearchFacetGroupHeader` and `CollectionFacetGroupHeader` below.